### DISCUSSION TITLE
Critical vulnerability in salt <= 3004.1

### DISCUSSION THREAD

**Discussion by TheRealPancakes** [Security-Onion-Solutions/securityonion#8125] [General] [answered]
Details are thin

[Notice](https://saltproject.io/security_announcements/attention-a-critical-vulnerability-has-been-discovered-in-salt-versions-3004-1-and-earlier/)

**Comment by TOoSmOotH** [comment, marked as answer]
We are monitoring closely and will release an update shortly after it is made available.

**Reply by dougburks** [reply]
Looks like the security issue only applies if you are using PAM authentication from within Salt which we do not by default:
https://blog.securityonion.net/2022/06/saltstack-security-issue.html
